cai_causal_graph/time_series_causal_graph.py

```diff
@@ -968,23 +968,31 @@ def from_adjacency_matrices(
         to X-1 -> X, where X is the set of nodes.
 
         Example:
+        >>> import numpy
+        >>> from cai_causal_graph import TimeSeriesCausalGraph
+        >>>
+        >>> # define the adjacency matrices
         >>> adjacency_matrices = {
-        ...     -2: numpy.array([[0, 0, 0], [1, 0, 0], [0, 0, 1]]),
-        ...     -1: numpy.array([[0, 1, 0], [1, 0, 0], [0, 0, 0]]),
-        ...     0: numpy.array([[0, 1, 1], [0, 0, 1], [0, 0, 0]]),
-        ... }
-
-        Let the nodes be X,Y,Z. The corresponding edges are:
-        >>> edges = [
-        ...     (Z-2, Y),
-        ...     (X-1, X),
-        ...     (Y-1, X),
-        ...     (Y-1, Y),
-        ...     (Z-1, Z),
-        ...     (X, Y),
-        ...     (X, Z),
-        ...     (Y, Z),
-        ... ]
+        >>>     -2: numpy.array([[0, 0, 0], [1, 0, 0], [0, 0, 1]]),
+        >>>     -1: numpy.array([[0, 1, 0], [1, 0, 0], [0, 0, 0]]),
+        >>>     0: numpy.array([[0, 1, 1], [0, 0, 1], [0, 0, 0]]),
+        >>> }
+        >>>
+        >>> # construct a time series causal graph from the adjacency matrices
+        >>> graph = TimeSeriesCausalGraph.from_adjacency_matrices(adjacency_matrices, variable_names=['X', 'Y', 'Z'])
+        >>>
+        >>> # query the edges of the constructed time series causal graph
+        >>> graph.get_edges()
+        >>> # output:
+        >>> # [
+        >>> #   Edge("X", "Y", type=->),
+        >>> #   Edge("X", "Z", type=->),
+        >>> #   Edge("X lag(n=1)", "Y", type=->),
```

Comment:
This doesn't match above. Was there an error before?

Reply:
Yes, it was incorrect before. I ran the adjacency matrix and used the output directly.

Reply:
Also note how there's no diagonal element in the `-1` adjacency matrix